recipes/unidist/meta.yaml

```diff
@@ -0,0 +1,133 @@
+{% set name = "unidist" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name }}
```

Comment:
```suggestion
  name: {{ name }}-meta
```

Reply:
Sorry, forgot that. Otherwise, there is a conflict with the actual `unidist` package. This one will not be available to end users, it is just needed for building.

Reply:
Yes, thanks! I added -packages suffix to be in align with some other packages (Modin, Ray, for example).